ckanext/switzerland/tests/test_dcat_helpers.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+import unittest
+import os
+
+import ckanext.switzerland.dcat.helpers as tk_dcat
+
+
+class TestDcatShaclHelpers(unittest.TestCase):
+
+    def setUp(self):
+        self.resultdir = tk_dcat.get_shacl_resultdir_from_config()
+        self.shacldir = tk_dcat.SHACL_SHAPES_DIR
+
+    def test_get_shacl_command_from_config(self):
+        result = tk_dcat.get_shacl_command_from_config()
+        self.assertEquals(result.split('/')[-1], 'shacl')
+
+    def test_get_shacl_resultdir_from_config(self):
+        result = tk_dcat.get_shacl_resultdir_from_config()
+        self.assertIsNotNone(result)
+
+    def test_get_shacl_shapedir(self):
+        result = tk_dcat.get_shacl_shapedir()
+        self.assertIsNotNone(result)
+
+    def test_get_shacl_data_file_path(self):
+        harvest_source_id = '93488cd9'
```

Comment:
any particular reason to not use a name that is a little more readable/distinguishable?

Reply:
yes, I used numbers, because they are ids in the real setup.